src/js/containers/covid19/Covid19Container.jsx

```diff
@@ -203,7 +225,9 @@ const Covid19Container = () => {
                     <GlobalModalContainer />
                 </main>
             </LoadingWrapper>
-            <Footer />
+            <div ref={lastSectionRef}>
```

Comment:
I think this might be creating this behavior?

The global should be taking up the whole width of the screen
![image](https://user-images.githubusercontent.com/12897813/88969657-2c0dc180-d27f-11ea-8db3-5a382317c644.png)